source/_integrations/nanoleaf.markdown

```diff
@@ -50,3 +51,58 @@ name:
 4. The output should include the auth token like *{"auth_token":"xxxxxxxxxxxxxxxxxxxxx"}*, copy the resulting token into your configuration
 
 If you get a 403 Forbidden message, you probably did not press the *ON* button long enough. The time-frame to get a valid token is only 30 seconds, so you have to be quick to issue the curl request.
+
+### Getting useful values for automations
+
+First set up some effects with the Nanoleaf app, and have them installed on the device.
+
+Once installed, go to "Developer Tools" and find the state of the device, eg `light.bedroom_triangles`. You should end up with something like this:
+
+```yaml
+# Example nanoleaf state:
+min_mireds: 154
+max_mireds: 833
+effect_list:
+  - Beatdrop
+  - Bedtime
+  - Blaze
+  - Cocoa Beach
+  - Cotton Candy
+  - Date Night
+  - Hip Hop
+  - Hot Sauce
+  - Jungle
+  - Lightscape
+  - Morning Sky
+  - Northern Lights
+  - Pop Rocks
+  - Prism
+  - Starlight
+  - Sundown
+  - Waterfall
+  - Vibrant Sunrise
+brightness: 155
+effect: Bedtime
+friendly_name: Triangles
+icon: 'mdi:triangle-outline'
+supported_features: 55
+```
+
+Now you can call the `light.turn_on` service with the data:
+```yaml
+# Example data to activate a nanoleaf effect:
+entity_id: light.trianglestest
+effect: Starlight
+```
+
+Or in a script for example:
+```yaml
+# Example alarm script
+alias: Alarm Sequence
+sequence:
+  - service: light.turn_on
+    data:
+      effect: Vibrant Sunrise
+    entity_id: light.triangles
+mode: single
+```
```

Comment:
same comment as above.

Reply:
I feel this is worth keeping, although I've added some extra commenting explaining why it exists, showing how we can use the effects that we discover from the state/UI.

Reply:
I don't agree. It is general light control, and not anything that is specific this integration. This does not belong on this page, as controlling a light is a generic thing, not a Nanoleaf thing.

Reply:
Ok, I have added a disclaimer noting the versions that were used for the examples, along with a link to the proper "Lights" integration page. This should direct users to the full docs for lights to find all the info that is available, but still give them something to get started with.